### DISCUSSION TITLE
Error when creating NPC via GUI

### DISCUSSION THREAD

**Discussion by bouatom** [cmu-sei/GHOSTS#416] [Q&A]
I'm trying to manually create an NPC but am running into a few errors. When I manually create an NPC I get this error message.
![image](https://github.com/user-attachments/assets/6d4b2655-ea9d-4462-8091-b1f3638d5ec5)
Error Code is 405

When I import from JSON, the GUI says it imported correctly but only the ID shows and nothing else is populated. I am copying a randomly generated NPC to test this functionality.

![image](https://github.com/user-attachments/assets/7c37dd9c-1b06-4cbe-b603-22e2d31cbcbf)

Ghost is running based on the documentation, found here: https://cmu-sei.github.io/GHOSTS/core/api/
Machine OS is ubuntu 22

**Comment by sei-dupdyke** [comment]
There is an endpoint missing to create pre-determined npcs. I added this in 8.2.1. Please update your api container to 8.2.1 and this should work. Sorry for the confusion and inconvenience.
